src/asn1acn.cpp

```diff
@@ -134,8 +134,7 @@ bool Asn1AcnPlugin::initialize(const QStringList &arguments, QString *errorStrin
 
     addAutoReleasedObject(new TypesLocator);
 
-    const auto directoryWatcher = new Libraries::ComponentDirectoryWatcher(
-        librariesSettings, std::make_unique<Libraries::CompontentLibraryDispatcher>());
+    const auto directoryWatcher = new Libraries::ComponentDirectoryWatcher(librariesSettings);
```

Comment:
doesn't it blocks directoryWatcher from ever being tested?

Reply:
_Ever_ is very strong word in this context :) I would say, that it makes it a little further from being testable, for the sake of being consistent. directoryWatcher had instance of QFileSystemWatcher created internally during initialization (this leaded me to conclusion, that it is wrong to have one object passed from the outside, and one created in the inside). Also, it already uses static LibraryStorage class internally (this leaded me to conclusion, that ComponentDirectoryWatcher might utilize static methods as well).